docs/docs/guides/preview/components/migrating-definitions.md

```diff
@@ -0,0 +1,58 @@
+---
+title: 'Migrating existing Definitions to components'
+sidebar_position: 350
+---
+
+:::note
+This guide covers migrating existing Python `Definitions` to components. This guide presupposes a components-enabled project. See the [getting started guide](./) or [Making an existing code location components-compatible](./existing-code-location) guide for more information.
```

Comment:
I think we have to migrate existing definitions, not `Definitions`. This guide is meant for people who have standalone dagster definitions.

Reply:
Ahh yeah, misunderstood. Newest version of the guide migrates plan-ol'-definitions